tests/integration/component.go

```diff
@@ -338,7 +338,18 @@ func componentTests(args ...string) {
 			helper.CmdShouldPass("odo", append(args, "create", "nodejs", "--project", project, "--env", "key=value,key1=value1")...)
 			helper.CmdShouldPass("odo", append(args, "push")...)
 			helper.CmdShouldPass("odo", append(args, "delete", "--all", "-f")...)
+			files := helper.ListFilesInDir(context)
+			Expect(files).NotTo(ContainElement(".odo"))
+		})
 
+		It("creates a local nodejs component without pushing and then deletes it using --all flag", func() {
+			helper.CopyExample(filepath.Join("source", "pyhton"), context)
+			helper.CmdShouldPass("odo", append(args, "create", "python", "--project", project, "--context", context)...)
+			helper.CmdShouldPass("odo", "push", "--context", context)
+			helper.CmdShouldPass("odo", "delete", "--context", context, "-f")
```

Comment:
Component name is missing

Reply:
helper.CmdShouldPass("odo", "delete", "--context", context, "-f") -> helper.CmdShouldPass("odo", append(args, "delete", "compNmae", "--context", context, "-f")...)

Reply:
Also just add one more validation step here as i mentioned in one of my comment
```
Verify the component name should not be present in ```odo component list```
```